.github/workflows/ci-emulator.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: Integration tests against emulator
+jobs:
+  units:
+    runs-on: ubuntu-latest
+
+    services:
+      emulator:
+        image: gcr.io/cloud-spanner-emulator/emulator:latest
+        ports:
+          - 9010:9010
+          - 9020:9020
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: stCarolas/setup-maven@v4
+      with:
+        maven-version: 3.8.1
```

Comment:
If we don't need a specific maven version we might want to avoid using third party actions: https://julienrenaux.fr/2019/12/20/github-actions-security-risk/. Not that we're passing in any secrets here though.

Reply:
Merges your suggestion. I just copied whatever jdbc testing did, I'd assume that setting a maven version would make the test more stable, but perhaps let's keep things simple until we need the complexity.